scrapinghub/client/jobs.py

```diff
@@ -77,6 +79,64 @@ def count(self, spider=None, state=None, has_tag=None, lacks_tag=None,
             params['spider'] = self.spider.name
         return next(self._project.jobq.apiget(('count',), params=params))
 
+    def cancel_jobs(self, keys=None, count=None, **params):
+        """Cancel a list of jobs using the keys provided.
+
+        :param keys: (optional) limit amount of returned jobs.
+        :param count: (optional) it requires admin access. Used for admins
+        to bulk cancel an amount of ``count`` jobs.
+
+        :return: a dict with the amount of jobs cancelled.
+        :rtype: :class:`dict`
+
+        Usage:
+
+        - cancel jobs 123 and 321 from project 111 and spiders 222 and 333::
+
+            >>> project.jobs.cancel_jobs(['111/222/123', '111/333/321'])
+            {'count': 2}
+
+        - cancel 100 jobs asynchronously::
+
+            >>> project.jobs.cancel_jobs(count=100)
+            {'count': 100}
+        """
+        update_kwargs(params, count=count, keys=keys)
+
+        if keys and count:
+            raise ValueError("keys and count can't be defined simultaneously")
+
+        elif not keys and not count:
+            raise ValueError("keys or count should be defined")
+
+        elif isinstance(keys, list) and len(keys) == 0:
```

Comment:
If keys can only be a `list` or `None` shouldn't this check be handled by `if not keys and not count`?

Reply:
+1, I think we can safely drop the condition, it's covered already